{{cookiecutter.repo_name}}/openedx.yaml

```diff
@@ -13,3 +13,5 @@ oeps:
     oep-5:
         state: False
         reason: TODO - Implement for this application if appropriate
+    oep-30:
+        state: True
```

Comment:
@edx/platform-team should the oep-30 state be True or False here?  This cookiecutter may or may not generate an app with full coverage; it depends on whether the user specifies models to auto-generate.

Reply:
I think that it's probably best to leave it `True` since that's both the most likely thing to happen, and the safest in terms or measuring which repos may have PII. IIRC there is a blurb of text that we can modify after the cookie cutter runs, if anything we might want to add a line there about updating this file for OEP-2 / OEP-30 compliance.

Reply:
Doesn't look like we already print a blurb of text after cookiecutting, but we do have a decent README which might be what you're remembering, so I updated that instead.